packages/react/src/components/TableHandles/TableCellHandle.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  DefaultInlineContentSchema,
+  DefaultStyleSchema,
+  InlineContentSchema,
+  StyleSchema,
+} from "@blocknote/core";
+import { ReactNode } from "react";
+
+import { createPortal } from "react-dom";
+import { MdArrowDropDown } from "react-icons/md";
+import { useComponentsContext } from "../../editor/ComponentsContext.js";
+import { TableCellHandleProps } from "./TableCellHandleProps.js";
+import { TableCellHandleMenu } from "./TableCellHandleMenu/TableCellHandleMenu.js";
+
+/**
+ * By default, the TableCellHandle component will render with the default icon.
+ * However, you can override the icon to render by passing children.
+ */
+export const TableCellHandle = <
```

Comment:
Not sure about the name `TableCellHandle`, I think handle implies that you can drag it around. Maybe smth like `TableCellOptions` would be better?